bot/exts/utilities/githubinfo.py

```diff
@@ -362,9 +356,78 @@ async def github_repo_info(self, ctx: commands.Context, *repo: str) -> None:
                 f"• Last Commit {last_pushed}"
             )
         )
+        return embed
 
-        await ctx.send(embed=embed)
 
+    @github_group.command(name="repository", aliases=("repo",))
+    async def github_repo_info(self, ctx: commands.Context, *repo: str) -> None:
+        """
+        Fetches a repository's GitHub information.
+
+        If the repository looks like `user/reponame` or `user reponame` then it will fetch it from github.
+        Otherwise, if it's a stored repo or PyDis repo, it will fetch the stored repo or use the PyDis repo
+        stored inside self.pydis_repos.
+        Otherwise it will fetch the most starred repo matching the search query from GitHub.
+        """
+        is_pydis = False
+        fetch_most_starred = False
+        repo_query = "/".join(repo)
+
+        if repo_query.count("/") > 1:
+            embed = discord.Embed(
+                colour=Colours.soft_red,
+                title=random.choice(NEGATIVE_REPLIES),
+                description="There cannot be more than one `/` in the repository."
+            )
```

Comment:
```suggestion
        if repo_query.count("/") > 1:
            embed = discord.Embed(
                title=random.choice(NEGATIVE_REPLIES),
                description="There cannot be more than one `/` in the repository.",
                colour=Colours.soft_red
            )
```
Very minor change, just keeping the code style consistent throughout